### ISSUE TITLE
found Images are not even to 1% Similar even though it shows 70%

### ISSUE THREAD

**Issue by bensh90** [Decimation/SmartImage#33]
I did multiple searches with images of people, and the results are always something entirely different.

I don't know if this is a bug, but I interpreted this tool as a Image Searching Tool no matter what the content of the picture is. Or can only specific Images be searched?

What I did was search for images with 1 or more People in it. What I found were scenes with nature like a view from a plane or a drone of a jungle or mountains. The images are not even closely similar, but the tool shows a similarity of 70% and higher.

**Comment by Decimation** [comment]
Which engines returned the result with 70%? The program itself retrieves results from various search engines -- it doesn't do the "searching" itself.

**Comment by bensh90** [comment]
Multiple ones. And like I said none of the pictures were even closely similar. I got better results searching for images directly on Google images or Yandex, Wich was why I was so surprised.

On engine seems to show various pictures on an anime Site. Other show some other random pictures.

If I have the time after the festivities, I will check with some images again

**Comment by Decimation** [comment]
You'll need to be more specific. 

Certain engines are specialized for types of images: *SauceNao*, *Iqdb*, *TraceMoe*, and *Ascii2D* are for anime and artwork specifically, and the similarity values may be false positives due to how each engine's algorithm works.

Furthermore, *SmartImage* does not do similarity calculations or the actual backend searching functionality for that matter; it aggregates the results returned by these various engines.

---

What I assume you're experiencing:

For example, if I upload a random image of a person to *TraceMoe*, I get false positives:
![image](https://user-images.githubusercontent.com/4624891/209445072-d867ab6a-9f0f-4470-8f9b-2bddb505e373.png)

If I run the same image with *SmartImage*:
![image](https://user-images.githubusercontent.com/4624891/209445195-ec0c6757-6b5c-44f8-a915-a60e44c28a58.png)


*RepostSleuth*, which searches Reddit results, is accurate. *TraceMoe*, which searches anime screenshots, returns false positives.

***To summarize: this is the correct behavior. *SmartImage* is working properly; it's just the engines returning false positives.***

**Comment by bensh90** [comment]
I see, thanks for clearing that up :)
